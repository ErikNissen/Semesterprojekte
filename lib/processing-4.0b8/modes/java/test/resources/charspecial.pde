version https://git-lfs.github.com/spec/v1
oid sha256:1137d9a4dd6996a946fcf0a6339ab941a289b166c92f693dd9d3bd300a18f04a
size 26
