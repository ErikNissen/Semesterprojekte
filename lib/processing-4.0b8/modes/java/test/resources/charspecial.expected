version https://git-lfs.github.com/spec/v1
oid sha256:27906148ea380c685389db8652c300aba8455f8544d5eb0a64f794e6c51b4803
size 728
